### PR TITLE
Added Xamarin Studio Cheat Sheet

### PR DESCRIPTION
**Pull request by gitcodes**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Added Xamarin Studio Cheat Sheet

---

Instant Answer Page: https://duck.co/ia/view/xamarin_cheat_sheet

---
https://duck.co/ia/view/xamarin_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/xamarin.json [hunk 1]
```diff
@@ -0,0 +1,263 @@
+{
+    "id": "xamarin_cheat_sheet",
+    "name": "Xamarin Studio",
+    "description": "Xamarin Studio Hotkeys",
+
+    "metadata": {
+        "sourceName": "xamarin",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/xamarin.json]: Could you uppercase this to `Xamarin`?

File: share/goodie/cheat_sheets/json/xamarin.json [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+        "sourceUrl" : "https://developer.xamarin.com/"
+    },
+
+    "aliases": [
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/xamarin.json]: Can remove this if there will be no aliases.

**Comment by gitcodes** [comment]
Specified changes are made.

File: share/goodie/cheat_sheets/json/xamarin.json [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+                "key": "[Ctrl][F5]",
+                "val": "Start Without Debugging"
+            },
+            {
+                "key": "[Ctrl][.]",
+                "val": "Show Fixes"
+            },
+              {
+                "key": "[Shift][F5]",
+                "val": "Stop Biuld execution"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/xamarin.json]: FIx the typo here

File: share/goodie/cheat_sheets/json/xamarin.json [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+        "File",
+        "Project",
+        "Search",
+        "Text Editor"
+    ],
+
+    "sections": {
+        "Navigation": [
+            {
+                "key": "[Shift][F12]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/xamarin.json]: @gitcodes please add a space ` ` between each key: `[Shift] [F12]`

**Comment by gitcodes** [comment]
added space b/w each keys .

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Xamarin Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/xamarin_cheat_sheet_mobile.png?U6Ncq78YLp)

**Comment by moollaza** [comment]
@gitcodes thanks for updating! This LGTM :+1:

**Comment by daxtheduck** [comment]
@gitcodes this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by edgesince84** [comment]
@gitcodes thanks for the fix!

**Pull request merged**